src/canister/individual_user_template/src/api/canister_lifecycle/post_upgrade.rs

```diff
@@ -37,6 +21,8 @@ fn post_upgrade() {
     restore_data_from_stable_memory();
     save_upgrade_args_to_memory();
     reenqueue_timers_for_pending_bet_outcomes();
+    // TODO: to be removed on the next updagrade
+    update_last_canister_functionality_access_time();
```

Comment:
lets not depend on `post_upgrade`. Lets add trigger functions in platform_orchestrator and user_index so that we are not stuck to remove this before running any other emergency upgrades.